includes/wf_crm_webform_base.inc

```diff
@@ -580,7 +580,8 @@ abstract class wf_crm_webform_base {
     static $status_types;
     static $membership_types;
     if (!isset($membership_types)) {
-      $membership_types = array_keys(wf_crm_apivalues('membershipType', 'get', ['is_active' => 1, 'return' => 'id']));
+      $domainId = CRM_Core_Config::domainID();
+      $membership_types = array_keys(wf_crm_apivalues('membershipType', 'get', ['is_active' => 1, 'domain_id' => $domainId, 'return' => 'id']));
```

Comment:
The goal of this module is to only use the CiviCRM API and not call internal Civi classes/functions such as `CRM_Core_Config::domainID()`. Fortunately in this case that's not necessary because you can pass the `current_domain` keyword to the api.

```suggestion
      $membership_types = array_keys(wf_crm_apivalues('membershipType', 'get', ['is_active' => 1, 'domain_id' => 'current_domain', 'return' => 'id']));
```